packages/legacy/core/App/navigators/defaultLayoutOptions.tsx

```diff
@@ -0,0 +1,8 @@
+import { ScreenLayoutConfig, Screens } from '../types/navigators'
+
+export const DefaultScreenLayoutOptions: ScreenLayoutConfig = {
+  [Screens.Terms]: {
+    customEdges: ['top', 'left', 'right'],
+  },
+  //TODO: Add more screens here
```

Comment:
It should be clear that adding new screens here requires submitting a new PR to include the wrapper at the screen level as well. 

However, I wonder if it might be more efficient to handle everything within this PR instead.

Reply:
I dont think it's a good idea to refactor all screens in this PR. Same as we did with History module, we implemented the feature and now Engineers from Quebec are picking up on it.